CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Dive pictures: support addition of videos
```

Comment:
i would change this to:
```
Dive media: support addition of videos
```

our base logic and classes should have been called "media" too.
because there was potential for videos even back when photos were first added.

Reply:
Done (also changed the other "Dive pictures" entry).